include/rmm/mr/device/hybrid_memory_resource.hpp

```diff
@@ -107,23 +110,6 @@ class hybrid_memory_resource : public device_memory_resource {
   LargeAllocMemoryResource* get_large_mr() { return large_mr_; }
 
  private:
-  /**
-   * @brief Get the memory resource to use to allocate the requested size.
-   *
-   * Returns the small memory_resource if `bytes` is <= the threshold size, or the large
-   * memory_resource otherwise.
-   *
-   * @param bytes Requested allocation size in bytes.
-   * @return rmm::mr::device_memory_resource& memory_resource that can allocate the requested size.
-   */
-  rmm::mr::device_memory_resource* get_resource(std::size_t bytes)
```

Comment:
This function should not be removed. The point of making it a function is to abstract and isolate the logic that chooses which resource to use.

Reply:
Added back.